LyokoAPI/VirtualEntities/LyokoWarrior/LyokoWarrior.cs

```diff
@@ -95,6 +95,14 @@ internal LyokoWarrior Xanafy()
             return this;
         }
 
+        internal LyokoWarrior Dexanafy()
```

Comment:
This is not a comment on your code but we might not have to do the location changes in these type of methods because IFSCL may call LW_MoveEvent. Since the rest of the file also manually changes the location though, you dont have to change this now.